src/components/modal/createAdmin/modules/NewAccount.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { Alert, Button, Stack, InputField, Text } from '@kiwicom/orbit-components/lib';
+import api from '../../../../../api';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+const NewAccount = ({ onHide, rerenderTable }) => {
+  const [alertTitle, setAlertTitle] = useState('');
+  const [showAlert, setShowAlert] = useState(false);
+  const [alertType, setAlertType] = useState('');
+  const [alertDescription, setAlertDescription] = useState('');
+
+  const displayAlert = (title, description, type) => {
+    setShowAlert(true);
+    setAlertTitle(title);
+    setAlertDescription(description);
+    setAlertType(type);
+  };
+
+  const handleFormSubmission = (values) => {
+    api.admins
+      .createWithNewAccount(values.email, values.password, values.name)
+      .then(() => {
+        onHide();
+        rerenderTable();
+      })
+      .catch((error) => {
+        console.error(error);
+        formik.setSubmitting(false);
+        displayAlert('Error', error.message, 'critical');
+      });
+  };
+
+  const validationSchema = Yup.object().shape({
+    email: Yup.string().email('Email must be a valid email').required('Required'),
+    name: Yup.string().required('Required'),
+    password: Yup.string()
+      .required('Required')
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[\!\@\#\$%\^&\*\-\+\=\?\;\:\[\]\{\}\|\`\~\"\'\_\.])(?=.{12,})/,
+        'Please create a password with at least 12 characters, comprising a mix of uppercase and lowercase letters, numbers and symbols'
+      ),
+    passwordConfirmation: Yup.string()
+      .required('Required')
+      .oneOf([Yup.ref('password'), null], 'Passwords must match'),
+  });
+
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      name: '',
+      password: '',
```

Comment:
the password field currently is not being used by the admins to login to the admin platform right? are we just saving the password just in case?

Reply:
oh it's bcos to create a new user account in firebase, have to set the password as well but it's not needed to login to the admin platform